src/core/client/stream/tabs/profile/queries/ProfileQuery.tsx

```diff
@@ -64,6 +64,10 @@ const ProfileQuery: StatelessComponent<InnerProps> = ({
       assetID,
       assetURL,
     }}
+    cacheConfig={{
+      // TODO: enable caching after mutations are adapted.
+      force: true,
+    }}
```

Comment:
What's this about?

Reply:
If we adapt the `CreateCommentMutation` to also insert it into the my profile list, we can activate caching here. We should however first integrate pagination for the my comments list first, before we can adapt them appropriately.